stream2hop/Utils.py

```diff
@@ -46,3 +49,47 @@ def add_common_arguments(parser):
         default=constant.CONFIG_FILE,
         help="hop client configuration file",
     )
+
+
+def get_secret(secret_name):
+    """
+        Get secret value from AWS
+        Args:
+            secret_name: secret name to fetch
+        returns:
+            secret value
+
+    """
+    region_name = "us-west-2"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager", region_name=region_name)
+
+    try:
+        get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+
+    except ClientError as e:
+        if e.response["Error"]["Code"] is not None:
+            raise e
+        else:
+            if "SecretString" in get_secret_value_response:
+                return get_secret_value_response["SecretString"]
+            else:
+                return base64.b64decode(get_secret_value_response["SecretBinary"])
```

Comment:
This block doesn't look right. You'll only be returning in the `except` case - that is, only if you had an error talking to AWS to get the secret.

Reply:
Ohh I am so sorry. Thanks